src/API/Site/Controllers/MerchantCenter/PriceBenchmarksController.php

```diff
@@ -96,15 +96,16 @@ protected function prepare_query_arguments( Request $request ): array {
 	protected function get_price_benchmarks_callback(): callable {
 		return function ( Request $request ) {
 			try {
-				$response_data = $this->get_price_benchmarks_base_data(
-					$this->prepare_query_arguments( $request )
-				);
+				/** @var PriceBenchmarks $price_benchmarks */
+				$price_benchmarks = $this->container->get( PriceBenchmarks::class );
+
+				$response_data = $price_benchmarks->get_price_benchmarks_data( $this->prepare_query_arguments( $request ) );
 
 				if ( empty( $response_data ) ) {
 					return new Response( [] );
 				}
```

Comment:
@mukeshpanchal27 @joemcgill very minor thing but is this check necessary? If the `$response_data` is empty is it not the same as the following line `return new Response( $response_data )`?

Reply:
Assuming the empty response is an empty array and not something like `null` or `false`, then yes. I think this just ensures we're sending the correct shape. The `PriceBenchmarks::get_price_benchmarks_data()` method has strict typing that it returns an array, so it's probably fine to remove this.